### ISSUE TITLE
Two breaking tests

### ISSUE THREAD

**Issue by kasperosterbye** [pharo-spec/NewTools-DocumentBrowser#3]
These two test break:
![image](https://user-images.githubusercontent.com/17723745/203491718-5fbb7e22-8048-4135-9bf6-1ed5a81c2656.png)

**Comment by kasperosterbye** [comment]
Something more fishy is going on.... investigating

**Comment by kasperosterbye** [comment]
The issue is that it catches an error in the version of Microdown included in Pharo11 (the integration branch).
When running Metacello load:
```smalltalk
Metacello new
 	 baseline: 'NewToolsDocumentBrowser';
 	 repository: 'github://pharo-spec/NewTools-DocumentBrowser:master/src';
 	 load.
```
It is recognised that Microdown is already loaded and the `master` of Microdown is not loaded. 
The document browser is depending on a few bugs in Microdown having been fixed.

So, in short this issue should go away once the current `master` of Microdown is merged into `integration`.

**Comment by MarcusDenker** [comment]
this is fixed
